ext/stackprof/stackprof.c

```diff
@@ -482,41 +497,61 @@ stackprof_record_sample_for_stack(int num, int timestamp_delta)
 	prev_frame = frame;
     }
 
-    if (_stackprof.raw) {
-	gettimeofday(&_stackprof.last_sample_at, NULL);
-    }
+    gettimeofday(&_stackprof.last_sample_at, NULL);
```

Comment:
(It seems simpler, and trivial, to assign this always, rather than track the downstream paths that do or don't need it.)